_source/logzio_collections/_prometheus-sources/dotnet-custom.md

```diff
@@ -0,0 +1,555 @@
+| Sum | [[your_histogram_name]]_sum |
+| Last Value | [[your_histogram_name]]_lastValue |
+| Max | [[your_histogram_name]]_max |
+| Mean | [[your_histogram_name]]_mean |
+| Median | [[your_histogram_name]]_median |
+| Min | [[your_histogram_name]]_min |
+| Percentile 75	| [[your_histogram_name]]_percentile75 |
+| Percentile 95 | [[your_histogram_name]]_percentile95 |
+| Percentile 98 | [[your_histogram_name]]_percentile98 |
+| Percentile 99 | [[your_histogram_name]]_percentile99 |
+| Percentile 999 | [[your_histogram_name]]_percentile999 |
+| Sample Size | [[your_histogram_name]]_sample_size |
+| Std Dev | [[your_histogram_name]]_std_dev |
+
+###### Timer
+
+| App Metrics label name | Logz.io label name |
+|---|---|
+| Duration Unit	 | duration_unit |
+| Rate Unit | rate_unit |
+
+| App Metrics parameter name | Logz.io parameter name |
+|---|---|
+| Count | [[your_timer_name]]_count |
+| Histogram Active Session | [[your_timer_name]]_histogram_active_session |
+| Histogram Sum | [[your_timer_name]]_histogram_sum |
+| Histogram Last Value | [[your_timer_name]]_histogram_lastValue |
+| Histogram Max	| [[your_timer_name]]_histogram_max |
+| Histogram Median | [[your_timer_name]]_histogram_median |
+| Histogram Percentile 75 | [[your_timer_name]]_histogram_percentile75 |
+| Histogram Percentile 95 | [[your_timer_name]]_histogram_percentile95 |
+| Histogram Percentile 98 | [[your_timer_name]]_histogram_percentile98 |
+| Histogram Percentile 99 | [[your_timer_name]]_histogram_percentile99 |
+| Histogram Percentile 999 | [[your_timer_name]]_histogram_percentile999 |
+| Histogram Sample Size | [[your_timer_name]]_histogram_sample_size |
+| Histogram Std Dev | [[your_timer_name]]_histogram_std_dev |
+| Rate One Min Rate | [[your_timer_name]]_rate_one_min_rate |
+| Rate Five Min Rate | [[your_timer_name]]_rate_five_min_rate |
+| Rate Fifteen Min Rate | [[your_timer_name]]_rate_fifteen_min_rate |
+| Rate Mean Rate | [[your_timer_name]]_rate_mean_rate |
+
+###### Apdex
+
+| App Metrics parameter name | Logz.io parameter name |
+|---|---|
+| Sample Size | [[your_apdex_name]]_sample_size |
+| Score | [[your_apdex_name]]_score |
+| Frustrating | [[your_apdex_name]]_frustrating |
+| Satisfied | [[your_apdex_name]]_satisfied |
+| Tolerating | [[your_apdex_name]]_tolerating |
+
+</div>
+
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="advanced-settings">
+
+#### Export using ToLogzioHttp exporter
+
+You can configure MetricsBuilder to use ToLogzioHttp exporter, which allows you to exprt metrics via HTTP using additional export settings. To enable this exporter, add the following code block to define the MetricsBuilder:
+
+```java
+var metrics = new MetricsBuilder()
+                .Report.ToLogzioHttp(options =>
+                {
+                    options.Logzio.EndpointUri = new Uri("[[logzio_endpoint]]");
+                    options.Logzio.Token = "[[metrics_token]]";
+                    options.FlushInterval = TimeSpan.FromSeconds(15);
```

Comment:
Please change [[logzio_endpoint]] and [[metrics_token]] to be like the variables in the explanation (<<LISTENER-HOST>>)